### ISSUE TITLE
Book of Horoscopes for 2020

### ISSUE THREAD

**Issue by robincamille** [NaNoGenMo/2019#84]
For every day in the new year, for every sign, a mangled horoscope reading. 

https://github.com/robincamille/nanogenmo2019 

The bulk of the text generation uses a Markov chain. I'm still working on it and deciding how nonsensical the horoscope can be. I also don't know much about astrology (this is meant as a whimsical project, I'm not trying to mock astrology), so I might let myself fall into a rabbithole and learn a bit more.

Here's a sample of my current output: 

### Aries

Argh, Aries! Which links and draws together the elements of the arts and music. Prosperity is in the desire placed on that one thing. Right now there's an energy pushing in.

### Taurus

Ptui, Taurus! Morning. Which may really mess things up. You will be a great opportunity. You are refusing offers of support. Are.

### Gemini

Aah, Gemini! You respect. The world may be an on-your-own-take-care-of-yourself-because-there-is-no-one-else-to-do-it-for-you grown-up. Today is the day after you die is a warm fire to keep the thing you want to do then do what you want, you'll get it. You have a yearning for perfection. You have an important role in your job. Stop it. You have exceeded what was expected. You have a friendly heart and are well liked by your delicate[sic] features. People find it difficult to resist persuasive.

### Cancer

Cheers, Cancer! Live a happy ending. Service to many places. You are focusing on what you want. That wasn't chicken. You will be mummy..

### Leo

No thanks, Leo! It all? you have a happier day. Don't let your mind set…confidence will lead you on. Go for the better. You are allowing your instincts guide you. Your good listening skills will open many doors. Two people shorten a road. All the troubles you have sense.

**Comment by greg-kennedy** [comment]
Did this get finished?  I'm looking at your repo and it's got this file, which may be the final text?
### https://github.com/robincamille/nanogenmo2019/blob/master/sample-output_all.md

**Comment by robincamille** [comment]
Yes, it was completed Dec. 3! I should have updated this issue. Thank you for checking it out.

GitHub cuts off the display of long markdown files, but if you download the file you linked to, you'll see the whole thing. For a shorter, display-friendly version of this project, here are the [horoscopes for just Leo in 2020](https://github.com/robincamille/nanogenmo2019/blob/master/sample-output_leo.md).

I had one output of this project printed and bound into a 501-page book, without any editing besides formatting:

<img src="http://robincamille.com/assets/nanogenmo2019-printed1.jpg" alt="Large book titled Book of Horoscopes 2020 on a wooden table"/>
<img src="http://robincamille.com/assets/nanogenmo2019-printed2.jpg" alt="Interior of large book with horoscope texts"/>

**Comment by hugovk** [comment]
Wow, looks great!

Please could you add a 50k+ example to the repo?

**Comment by robincamille** [comment]
Done: https://raw.githubusercontent.com/robincamille/nanogenmo2019/master/sample-output_all_full.md (200k words)

Thanks for organizing this, @hugovk!

**Comment by hugovk** [comment]
And thank you for entering! Have a "completed" label!

What service did you use for printing the book?
